src/server_manager/web_app/ui_components/cloud-install-styles.js

```diff
@@ -1,23 +1,7 @@
-<!--
-  Copyright 2018 The Outline Authors
-
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
+import '@polymer/polymer/polymer-legacy.js';
```

Comment:
Changed to follow the instructions here: https://polymer-library.polymer-project.org/3.0/docs/devguide/style-shadow-dom#share-styles-between-elements.

Notice that this is a "style module", not a "polymer element".

Reply:
Changed to follow the instructions at https://polymer-library.polymer-project.org/3.0/docs/devguide/style-shadow-dom#share-styles-between-elements

I'd rather avoid those other changes in this PR, since it's huge. Less things to possibly reconcile.